DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -45,7 +45,7 @@ int tutorialId
                 // Using this list of other tutorials in the course we can work out if there is another item in the

```

Comment:
Should we also move this to a dataservice & rename?

Reply:
Yep. Was going to do them in separate MRs. Should I do it in this one instead?

Reply:
Feel free to do a separate one if that's the plan (I would have been happy either way) - will approve this 👍